src/global/a11y/index.ts

```diff
@@ -0,0 +1,3 @@
+export * from './arrow-navigation';
```

Comment:
for me it's not really clear why those files belong to a11y. focus and arrow navigation is a standard for me and nothing special a11y related. and assign-id is often used to help reference something, but also for me for itself has nothing to do with a11y. hmm :D